menas/src/main/scala/za/co/absa/enceladus/menas/WebSecurityConfig.scala

```diff
@@ -72,6 +72,9 @@ class WebSecurityConfig @Autowired()(beanFactory: BeanFactory,
         .and()
           .addFilterBefore(kerberosFilter, classOf[UsernamePasswordAuthenticationFilter])
           .addFilterAfter(jwtAuthFilter, classOf[SpnegoAuthenticationProcessingFilter])
+        .headers()
```

Comment:
I think this will also be removed in the separation PR

Reply:
I can see that. And as it's nor crucial now, I will remove it.